tensorboard/plugins/base_plugin.py

```diff
@@ -123,9 +127,12 @@ def __init__(
           mapping, lest a KeyError is raised.
       window_title: A string specifying the the window title.
     """
+    assert not (db_uri and logdir)
```

Comment:
An assertion won't generate a very user-friendly failure?

Reply:
Changed to raising `ValueError`s.